packages/components/molecules/f-restaurant-card/src/components/subcomponents/IconText.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+    <div>
+        <p
+            :class="{
+                [$style['c-restaurantCard-iconText']]: true,
+                [$style['c-restaurantCard-iconText--bold']]: isBold
+            }"
+            aria-hidden="true"
+            :data-test-id="dataTestId">
+            <span
+                v-if="showIcon"
+                :class="{
+                    [$style['c-restaurantCard-iconText-icon']]: true,
+                    [$style['c-restaurantCard-iconText-icon--hideOnMidBellow']]: hideIconInTileView
+                }"
+                :style="`fill:${color};`"
+                data-test-id="icon-text-icon">
+                <slot />
+            </span>
+            <span
+                :class="$style['c-restaurantCard-iconText-content']"
+                :style="`color:${color};`"
+                data-test-id="icon-text-visible-text">
+                {{ text }}
+            </span>
+        </p>
+        <span
+            class="is-visuallyHidden"
+            data-test-id="icon-text-accessible-text">
+            {{ accessibleText || text }}
+        </span>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'IconText',
+    props: {
+        text: {
+            type: String,
+            required: true
+        },
+        accessibleText: {
+            type: String,
+            default: null
```

Comment:
Also unsure if this is worthwhile if it just duplicates the regular text prop when no accessible string is provided